spec/controllers/miq_report_controller/dashboards_spec.rb

```diff
@@ -5,7 +5,9 @@
     let(:user)           { FactoryBot.create(:user, :features => "db_edit") }
 
     before do
-      stub_user(:features => :all)
+      EvmSpecHelper.seed_specific_product_features(%w(db_edit db_copy))
+      EvmSpecHelper.local_miq_server
+      login_as user
```

Comment:
The UI specs have a helper methods for setting up a user with specific features all in one go: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/spec/support/controller_helper.rb#L25-L28 .  Not sure if that's something we should leverage here or not.

Reply:
DONE